daisy_integration_tests/scripts/post_translate_test.ps1

```diff
@@ -45,16 +71,29 @@ function Check-Activation  {
   }
 }
 
+function Check-SkipActivation {
+  if (!((Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform').KeyManagementServiceName -eq $null)) {
```

Comment:
How about:

$kms = (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform').KeyManagementServiceName

if ($kms -ne "") {
  throw "KMS server set when it should be blank: $kms"
}

Reply:
We are not testing for an empty string, but rather the null value.